yb-voyager/cmd/analyzeSchema.go

```diff
@@ -259,6 +260,29 @@ func reportSummary() {
 	}
 }
 
+func addSummaryDetailsForIndexes() {
+	var indexesInfo []utils.IndexInfo
+	found, err := metaDB.GetJsonObject(nil, metadb.SOURCES_INDEXES_INFO_KEY, &indexesInfo)
+	if err != nil {
+		utils.ErrExit("analyze schema report summary: load indexes info: %s", err)
+	}
+	if found {
+		exportedIndexes := lo.Keys(summaryMap["INDEX"].objSet)
+		unexportedIdxsMsg := "Indexes which are neither exported by yb-voyager as they are unsupported in YB and needs to be handled manually:\n"
+		unexportedIdxsPresent := false
+		for _, indexInfo := range indexesInfo {
+			sourceIdxName := indexInfo.TableName + "_" + strings.Join(indexInfo.Columns, "_")
```

Comment:
Is there any constraint on the order of columns in the generated index name?

Reply:
yes, I confirmed that ora2pg follows the same order in which this query returns the column names.

Reply:
beacuse of `WITHIN GROUP (ORDER BY AIC.COLUMN_POSITION)` in query we are using:
```
query := fmt.Sprintf(`SELECT AIN.INDEX_NAME, AIN.INDEX_TYPE, AIN.TABLE_NAME, 
	LISTAGG(AIC.COLUMN_NAME, ', ') WITHIN GROUP (ORDER BY AIC.COLUMN_POSITION) AS COLUMNS
	FROM ALL_INDEXES AIN
	INNER JOIN ALL_IND_COLUMNS AIC ON AIN.INDEX_NAME = AIC.INDEX_NAME AND AIN.TABLE_NAME = AIC.TABLE_NAME
	LEFT JOIN ALL_CONSTRAINTS AC 
	ON AIN.TABLE_NAME = AC.TABLE_NAME 
	AND AIN.INDEX_NAME = AC.CONSTRAINT_NAME
	WHERE AIN.OWNER = '%s' 
	AND NOT (AIN.INDEX_NAME LIKE 'SYS%%' OR AIN.INDEX_NAME LIKE 'DR$%%')
	AND AC.CONSTRAINT_TYPE IS NULL -- Exclude primary keys
	GROUP BY AIN.INDEX_NAME, AIN.INDEX_TYPE, AIN.TABLE_NAME`, ora.source.Schema)
```

same is being followed by ora2pg